flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java

```diff
@@ -960,14 +960,15 @@ public int parseParameters(String[] args) {
 		
 		// check for action
 		if (args.length < 1) {
-			System.out.println("Please specify an action.");
 			printHelp();
+			System.out.println();
+			System.out.println("Please specify an action.");
```

Comment:
instead of adding an additional call each time for the new-line, why aren't you pre-pending a '\n' to the string?